src/commands/command-getkeys.json

```diff
@@ -14,6 +14,17 @@
         ],
         "acl_categories": [
             "CONNECTION"
+        ],
+        "arguments": [
+            {
+                "name": "command",
+                "type": "string"
+            },
+            {
+                "name": "arg",
+                "type": "string",
+                "multiple": true
```

Comment:
i think `optional` is needed too.
doing `COMMAND GETKEYS PING` is valid.

Reply:
`COMMAND GETKEYS`'s arity is -4, so it's invalid.

Reply:
so that's a regression in v7.0.
in the past it was
```c
    } else if (!strcasecmp(c->argv[1]->ptr,"getkeys") && c->argc >= 3) {
```

Reply:
So should we update it?